docs/source/overview/prerequisites.rst

```diff
@@ -18,35 +18,47 @@ This is a standard example:
   [paths]
   # inputs
   grids = "data/grids"
-  theory_cards = "data/theory_cards"
   operator_card_template_name = "_template.yaml"
   # outputs
   operator_cards = "data/operator_cards"
   ekos = "data/ekos"
   fktables = "data/fktables"
 
+  # The following two keys are only necessary when nnpdf=false
+  #theory_cards = "data/theory_cards"
+  #ymldb = "data/yamldb"
+
   [paths.logs]
   eko = "logs/eko"
   fk = "logs/fk"
 
 
 All the relevant inputs are described below. The command ``pineko scaffold new`` will generate all necessary folders.
 
+
+Theory Runcards
+---------------
+
+You need to provide the necessary theory runcards named with their respective theory ID inside the *paths.theory_cards* folder [1]_ unless you are using the ``nnpdf`` key (see below).
+For more details about theory runcards you can look at https://eko.readthedocs.io/en/latest/code/IO.html under **Theory Runcards**.
```

Comment:
I'm not sure how much that link is up-to-date (the answer is it isn't) and it is surely not the official source of truth of which we are speaking here - instead it is NNPDF, i.e. https://docs.nnpdf.science/theory/theoryparamsdefinitions.html should be linked instead